doc/alignment.md

```diff
@@ -0,0 +1,61 @@
+
+Partition Alignment
+===================
+
+Unfortunately these days for optimal performance care must be taken that
+partitions are properly aligned, see e.g.
+https://en.wikipedia.org/wiki/Partition_alignment.
```

Comment:
It's very good to have a link here, statements like this have to be supported by more info, such as articles, research etc. Nice.

I just failed to find an exact part in that article (and those linked ones) that would explain that statement. Maybe I was not looking at the right place.